src/gc/handletable.cpp

```diff
@@ -301,7 +301,7 @@ OBJECTHANDLE HndCreateHandle(HHANDLETABLE hTable, uint32_t uType, OBJECTREF obje
     }
     CONTRACTL_END;
 
-#if defined( _DEBUG) && !defined(FEATURE_REDHAWK)
+#if defined( _DEBUG) && !defined(FEATURE_REDHAWK) && !defined(BUILD_AS_STANDALONE)
```

Comment:
It would be better to delete the inject faults from here, and move them to the EE side instead.

Reply:
does anyone actually use this? I have not seen it myself

---
In reply to: [184080041](https://github.com/dotnet/coreclr/pull/17769#discussion_r184080041) [](ancestors = 184080041)

Reply:
It has been used for SQL testing 10+ years ago. It is not actively used for anything. I guess we can just delete it.

Reply:
yeah, I vote for deleting it.

Reply:
Ok, it's deleted now